include/psa/crypto_builtin_cipher.h

```diff
@@ -41,7 +41,10 @@ typedef struct {
     psa_algorithm_t alg;
     uint8_t iv_length;
     uint8_t block_length;
-    mbedtls_cipher_context_t cipher;
+    union {
+        unsigned int initialised;
```

Comment:
I understand why you do that but that is not very elegant I'd say. Why not having a proper initialization macro for `cipher` instead ?

Reply:
I share your opinion about it is not so elegant. There were a discussion in slack about this and the outcome was to use a union because `PSA_CIPHER_OPERATION_INIT` doesn't need to initialize a `mbedtls_cipher_context_t` structure.